sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -417,7 +425,15 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
     this.educationProgramOfferingControllerService.assertOfferingsValidationsAreValid(
       offeringValidations,
       csvModels,
+      validationOnly,
     );
+
+    if (validationOnly) {
+      // If the endpoint is called only to perform the validation and no error was found
+      // return an empty array because no record will be created.
+      return [];
+    }
```

Comment:
From the comment on line 432, it seems you missed taking error(s) found into consideration before returning an empty array.

Reply:
If any error was found the code will not reach this point.
So we have two conditions in the comments:
1 - "endpoint is called only to perform the validation";
2 - "and no error was found";
then return an empty array.
Maybe I am not capturing the recommendation. Can you please share an example to improve the comment?

Reply:
IMO the comment is 100% okay. The part of it the code not reaching line 431 if an error is found makes me understand it much better. Thanks for the explanation, I will resolve.